src/lucky/redirectable.cr

```diff
@@ -45,12 +45,12 @@ module Lucky::Redirectable
   # Redirect to the given path, with a human friendly status
   #
   # ```crystal
-  # redirect to: "/users", status: Status::MovedPermanently
+  # redirect to: "/users", status: HTTP::Status::MOVED_PERMANENTLY
```

Comment:
Thoughts on changing these docs to use the symbol version?

Reply:
I think that's fine. The guides could mention that these map to the HTTP::Status and link to those docs, and just teach people to use the symbols instead. I'll remove all of the enum constants and replace with symbols.